src/inc/quic_datapath.h

```diff
@@ -639,7 +639,8 @@ CxPlatSocketSend(
     _In_ CXPLAT_SOCKET* Socket,
     _In_ const CXPLAT_ROUTE* Route,
     _In_ CXPLAT_SEND_DATA* SendData,
-    _In_ uint16_t PartitionId
+    _In_ uint16_t PartitionId,
+    _In_ BOOLEAN InlineHint
```

Comment:
Doing it this way seems to be extremely disruptive. What if, instead, we defined:
```c
_IRQL_requires_max_(DISPATCH_LEVEL)
void
CxPlatSendDataSetInlineHint(
    _In_ CXPLAT_SEND_DATA* SendData,
    _In_ BOOLEAN SendInline
    );
```
And then, for datapaths that support this, it saves it on the `CXPLAT_SEND_DATA` internally, and uses the state when the app later calls `CxPlatSocketSend`.

Reply:
Yeah, I thought about this but went with a more difficult way..